lib/libvarnish/vcli_serve.c

```diff
@@ -93,9 +95,14 @@ enum cmd_error_e {
 void v_matchproto_(cli_func_t)
 VCLS_func_close(struct cli *cli, const char *const *av, void *priv)
 {
+	struct VCLS *cs;
 
 	(void)av;
 	(void)priv;
+	cs = cli->cls;
+	CHECK_OBJ_NOTNULL(cs, VCLS_MAGIC);
```

Comment:
can we use (something like) `ASSERT_CLI()`?

Reply:
This is `libvarnish`, so not really.

Reply:
Or I guess we could have something like `CHECK_VCLS(cs)` in this file that performs a magic check followed by a thread check?

Reply:
Added `CHECK_VCLS` macro in the latest revision